docs/modules/wan/pages/tuning.adoc

```diff
@@ -800,3 +800,84 @@ hazelcast:
 ----
 ====
 You can find the specifications of these properties in xref:ROOT:system-properties.adoc[System Properties].
+
+== Static Endpoints Discovery Strategy Health Checking Properties
+
+Since its introduction in Hazelcast 5.4, the default behaviour is for the static endpoint discovery strategy
+to perform health checks on endpoints. There is no additional configuration required beyond defining the static
+endpoints as our default values cover most use-cases, however the following properties can be adjusted to suit
+specific needs:
+
+* `health-check-interval-ms` (default: `10000`): Defines the interval (in milliseconds) between connection health check tasks.
+Consider that health checks involve attempting connections which may take up to 10 seconds to timeout per connection, so tasks
+should be adequately spaced apart as to lower system resource usage.
+* `health-check-timeout-ms` (default: `3000`): Defines the maximum duration to wait (in milliseconds) when checking connection health.
+* `health-check-max-failed-probes` (default: `10`):  When an unhealthy connection is not found during target discovery, it is
+marked as a "failed probe". This property defines the maximum number of times a connection probe can be marked as failed in a row,
```

Comment:
There is not, I suppose it was somewhat there because I haven't explicitly defined what a failed probe is, but I think the context is clear enough. Remove the quotes: https://github.com/hazelcast/hz-docs/pull/972/commits/25691da16fcdc76a54688eb0983e54427485c07f